src/tools/auth0/handlers/pages.ts

```diff
@@ -41,6 +41,7 @@ export default class PagesHandler extends DefaultHandler {
     super({
       ...options,
       type: 'pages',
+      identifiers: ['name'],
```

Comment:
By adding an identifier field here, we are able to enable keyword preservation for pages.